packages/dids/tests/methods/did-key.spec.ts

```diff
@@ -276,12 +277,7 @@ describe('DidKey', () => {
         ],
       };
 
-      try {
-        await DidKey.getSigningMethod({ didDocument });
-        expect.fail('Error should have been thrown');
-      } catch (error: any) {
-        expect(error.message).to.equal('Method not supported: example');
-      }
+      await expect(DidKey.getSigningMethod({ didDocument })).to.be.rejectedWith('Method not supported: example');
```

Comment:
revert to using try / catch and not `chaiAsPromised`

```suggestion
      try {
        await DidKey.getSigningMethod({ didDocument });
        expect.fail('Error should have been thrown');
      } catch (error: any) {
        expect(error.message).to.equal('Method not supported: example');
      }
```